samples/matter/light_switch/src/binding_handler.cpp

```diff
@@ -0,0 +1,259 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
```

Comment:
The copyright header is wrong, in NCS it should be something like this:

/*
 * Copyright (c) 2022 Nordic Semiconductor ASA
 *
 * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 */

Reply:
Please fix this in all added files (not including this auto-generated).